Libraries/ActionSheetIOS/RCTMessageActivityItemProvider.m

```diff
@@ -0,0 +1,28 @@
+//
+//  RCTMessageActivityItemProvider.m
+//  RCTActionSheet
+//
+//  Created by Poon Kwok Cheung on 11/7/2017.
+//  Copyright © 2017 Facebook. All rights reserved.
+//
+
+#import "RCTMessageActivityItemProvider.h"
+
+@implementation RCTMessageActivityItemProvider
+
+
+- (id) activityViewController:(UIActivityViewController *)activityViewController itemForActivityType:(NSString *)activityType
+{
+  if([activityType isEqualToString:@"net.whatsapp.WhatsApp.ShareExtension"]) {
```

Comment:
Can you add a space after `if` here? You can look at [other files](https://github.com/facebook/react-native/blob/master/Libraries/ActionSheetIOS/RCTActionSheetManager.m) in the repo to get an idea of the coding style we use.